packages/analytics-js-integrations/src/integrations/DCMFloodlight/utils.js

```diff
@@ -99,7 +99,7 @@ const mapFlagValue = (key, value) => {
     return 0;
   }
 
-  throw Error(`[DCM Floodlight]:: ${key}: valid parameters are [1|true] or [0|false]`);
+  return 0;
```

Comment:
does this mean we are not offering any visibility to the user about why the event is dropped?

Reply:
we can use logger.error to show error message but we can't throw an exception to core SDK. in this case we can always fall back to default value (which is 0) so there is no need to throw an error